apps/rails/app/models/user_role.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class UserRole < ApplicationRecord
+  belongs_to :user
+  belongs_to :role
+  belongs_to :access_area_scope, optiona: true
```

Comment:
just a letter missing

```suggestion
  belongs_to :access_area_scope, optional: true
```

Reply:
why `access_area_scope` it's optional? we'll be going to have roles that manage all the access areas?